src/resources/i18n/translations/en.json

```
@@ -7,17 +7,17 @@
 	"ARIA HINT - Error something went wrong with playing this item": "Could not play this.",
 	"ARIA HINT - Hide podcast settings": "Hide podcast settings",
 	"ARIA HINT - Max playback speed": "Set max playback speed for your player",
-	"ARIA HINT - Membership - Create and share clips": "Create and share clips with Podverse premium.",
-	"ARIA HINT - Membership - Create playlists": "Create playlists with Podverse premium.",
-	"ARIA HINT - Membership - Download a backup of your data": "Download a backup of your data with Podverse premium.",
+	"ARIA HINT - Membership - Create and share clips": "Create and share clips with Podverse Premium.",
+	"ARIA HINT - Membership - Create playlists": "Create playlists with Podverse Premium.",
+	"ARIA HINT - Membership - Download a backup of your data": "Download a backup of your data with Podverse Premium.",
 	"ARIA HINT - Membership - Download episodes": "Download episodes.",
 	"ARIA HINT - Membership - Drag and drop queue": "Player queue.",
 	"ARIA HINT - Membership - Sleep timer": "Sleep timer.",
 	"ARIA HINT - Membership - Subscribe to podcasts": "Subscribe to podcasts.",
-	"ARIA HINT - Membership - Support free and open source software": "Support copylefted libre software 😊",
-	"ARIA HINT - Membership - Sync your queue on all devices": "Sync your queue to all devices with Podverse premium.",
-	"ARIA HINT - Membership - Sync your subscriptions on all devices": "Sync your subscriptions to all devices with Podverse premium.",
-	"ARIA HINT - Membership features header": "Features the app presented as gratis, and premium features.",
+	"ARIA HINT - Membership - Support free and open source software": "Support free and open source software 😊",
+	"ARIA HINT - Membership - Sync your queue on all devices": "Sync your queue to all devices with Podverse Premium.",
+	"ARIA HINT - Membership - Sync your subscriptions on all devices": "Sync your subscriptions to all devices with Podverse Premium.",
+	"ARIA HINT - Membership features header": "Features the app provides for free, and Premium features.",
```

```suggestion
	"ARIA HINT - Membership features header": "Features the app provides gratis, and Podverse Premium features.",
```
"free" blurs the lines between free as in freedom/free software, and gratis.
Mentally this focus on price takes away from the main selling point to set it apart from freeware.

---

👍